x-pack/plugins/elastic_assistant/server/routes/knowledge_base/entries/create_route.ts

```diff
@@ -18,12 +18,12 @@ import {
 } from '@kbn/elastic-assistant-common/impl/schemas/knowledge_base/common_attributes.gen';
 import { ElasticAssistantPluginRouter } from '../../../types';
 import { buildResponse } from '../../utils';
-import { UPGRADE_LICENSE_MESSAGE, hasAIAssistantLicense } from '../../helpers';
+import { performChecks } from '../../helpers';
 
 export const createKnowledgeBaseEntryRoute = (router: ElasticAssistantPluginRouter): void => {
   router.versioned
     .post({
-      access: 'public',
+      access: 'internal',
```

Comment:
This route was not previously registered, but it is now as of this PR, so marking as `internal` until we're ready.